core/sparse_array.h

```diff
@@ -57,12 +57,21 @@ class SparseArray
 	
 	uint32_t nrows_, ncols_;		// the number of rows and columns; determined at construction time
 	uint32_t nrows_set_;			// the number of rows that have been configured (at least partially, during building)
-	uint32_t nnz_;					// the number of non-zero entries in the sparse array (also at row_offsets[nrows_set])
 	uint32_t nnz_capacity_;			// the number of non-zero entries allocated for at present
 	
 	bool finished_;					// if true, Finished() has been called and the sparse array is ready to use
+
+
+	uint32_t nnz_;					// the number of non-zero entries in the sparse array (also at row_offsets[nrows_set])
```

Comment:
Seems like this line was better where it used to be, above...?

Reply:
Done!